client/me/connected-application-icon/index.jsx

```diff
@@ -6,17 +6,7 @@ import './style.scss';
 export default class extends Component {
 	static displayName = 'ConnectedApplicationIcon';
 
-	static defaultProps = {
-		size: 40,
-	};
-
 	render() {
-		return (
-			<PluginIcon
-				className="connected-application-icon"
-				image={ this.props.image }
-				size={ this.props.size }
```

Comment:
Is the size property not needed anymore?
I'm just curious how it was used previously and why it's not used anymore.

Reply:
It looks like `PluginIcon` support for `size` may have been removed prior to the [initial Calypso commit](https://github.com/Automattic/wp-calypso/blob/53bde8c3f9c4668e62dbd2c670aaacc2bd820427/client/my-sites/plugins/plugin-icon/plugin-icon.jsx).